account-integrations/webauthn/src/lib/FCL_Webauthn.sol

```diff
@@ -28,13 +28,14 @@ library FCL_WebAuthn {
     error InvalidClientData();
     error InvalidSignature();
 
+    /** @notice Modified from original Fresh Crypto Lib code to use memory instead of calldata */
```

Comment:
Curious why this change is needed? PS can appreciate that arrays/bytes can be a pain regarding access.

Reply:
If referring to using `memory` instead of `calldata`, answered in other comment and PR description. Let me know if what I'm saying doesn't makes sense